### DISCUSSION TITLE
Migrate to Avalonia 11?

### DISCUSSION THREAD

**Discussion by danipen** [AvaloniaCommunity/Material.Avalonia#235] [Q&A] [answered]
Are there plans for migrating to Avalonia 11 RC?

**Comment by SKProCH** [comment, marked as answer]
Yes, of course, some parts already made in `avalonia-11-rc1` branch, i'll plan to complete this today (i hope)

**Reply by SKProCH** [reply]
@danipen i've published a new version on nuget - `3.0.0-avalonia11-preview4.179-nightly`. It has some temporary workarounds, for example some custom animation would not work, but i plan to return it asap.

**Reply by danipen** [reply]
Ok. Thanks. Please let me know when the animations bring to life again.

**Reply by danipen** [reply]
Any progress on this? I guess ... no. Are you blocked by avalonia hiding their API or is a lack of time?

**Reply by SKProCH** [reply]
Seems like most of the things should work for now. I brought animations back but also figured out what we actually almost don't use it. Currently we have a bug with runtime theme changing, which will be fixed as the new avalonia version will be released (or you can use nightly).
Answering your question I blocked mostly by the time. 

Also, I plan to release a new preview when I fix this - https://github.com/AvaloniaCommunity/Material.Avalonia/issues/241

**Reply by danipen** [reply]
I'm asking specifically on this: https://github.com/AvaloniaCommunity/Material.Avalonia/blob/cc241ae984a76b81cfc40e183ba56c8daf9053f1/Material.Styles/Controls/Rotator.cs#L63

**Reply by SKProCH** [reply]
In the next Avalonia rc will be a new API for this, but for now it is implemented with a workaround and works properly.
